lib/connection.js

```diff
@@ -63,6 +64,10 @@ Connection.prototype.connectMethod = function (config, callback) {
   }
 }
 
+Connection.prototype.connectReplicaMethod = function (config, callback) {
+  this.connectRedis(config.redisReplicaUrl, callback)
```

Comment:
**minor pick:** we could add an extra parameter here to differentiate when is a replica redis calling the method connectRedis , this way the log `Redis client "ready" event.` could be changed to Redis replica client "ready" event.